_zh-cn/overviews/scala3-book/introduction.md

```diff
@@ -5,26 +5,25 @@ description: This page begins the overview documentation of the Scala 3 language
 num: 1
 previous-page: 
 next-page: scala-features
-
-scala3: true
-partof: scala3-book
-overview-name: "Scala 3 — Book"
-layout: multipage-overview
-permalink: "/zh-cn/scala3/book/:title.html"
```

Comment:
This permalink needs to be applied across the whole translated book for it to render correctly

Reply:
add.